app/validators/appeal_details/answers_validator.rb

```diff
@@ -18,6 +18,7 @@ def validate
     end
 
     def complete?
+      return true if record.cifc?
```

Comment:
Would it be better to add this guard to the '#applicable?' method?

Reply:
@mseedat-moj Is this correct? as the `appeal_original_app_submitted` question could have been answered in a cifc application?